src/index.css

```diff
@@ -38,27 +38,28 @@ code {
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background: #fafafa;
+  background-color: #fafafa;
   margin-top: 8.3rem;
+  width: 87.5rem;
```

Comment:
- [ ] On my laptop the design is stretching too much and is not looking good, kindly check into this, it is never a good idea to make a fixed general width because remember users can check it from laptop, dekstop with different sizes
![Uploading image.png…]()